package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "fintegrate-auth",
-  "version": "1.0.28",
+  "version": "1.0.113",
```

Comment:
עדיף להשתמש בגרסת beta כשרוצים לבדוק את הקוד, ולא להעלות גרסאות שלא מספיק עובדות
https://kevinkreuzer.medium.com/publishing-a-beta-or-alpha-version-to-npm-46035b630dd7

Reply:
עבדתי עם זה, אך הייתה לי שגיאה שלא קשורה לזה, ולכן ביטלתי כרגע את השימוש ב beta, משלב זה והלאה - נעלה גרסאות ניסיון ב beta.